common/src/main/java/io/novafoundation/nova/common/utils/coroutines/RootScope.kt

```diff
@@ -3,4 +3,5 @@ package io.novafoundation.nova.common.utils.coroutines
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.MainScope
 
+@Deprecated("Use scope in RootViewModel instead")
```

Comment:
I think it would also be great to explain deprecation so we wont forget about reasons. Something like 
"Using RootScope might have unintended side effects in case RootViewModel, where most of the clients of RootScope are used, is re-created - RootScope jobs wont be cancelled leading to potentially duplicated jobs"